packages/nuxt/src/app/components/nuxt-link.ts

```diff
@@ -207,6 +249,7 @@ export function defineNuxtLink (options: NuxtLinkOptions) {
         required: false,
       },
     },
+    useLink: (props: UseLinkOptions) => useNuxtLink(props as NuxtLinkProps),
     setup (props, { slots }) {
```

Comment:
ideally we should not only expose this but also use `useLink` internally within the setup function, to reduce code duplication and ensure the logic remains the same

Reply:
yeah, check linked vuetify PR: https://github.com/vuetifyjs/vuetify/pull/19514